sdk/iot/modelsrepository/tsconfig.json

```diff
@@ -0,0 +1,16 @@
+{
+  "extends": "../../../tsconfig.package",
+  "compilerOptions": {
+    "outDir": "./dist-esm",
+    "declarationDir": "./types",
+    "lib": ["dom", "esnext"],
```

Comment:
will just compile against dom

Reply:
I'm curious what shapes you need from dom, since sometimes node-only customers complain about this dependency so we try to avoid it: https://azure.github.io/azure-sdk/typescript_design.html#ts-config-lib

Reply:
gotcha I was basing leaving dom in off another SDK in the repo. Will remove.